sia-antfarm/ant.go

```diff
@@ -10,14 +10,15 @@ import (
 	"time"
 
 	"github.com/NebulousLabs/Sia/api"
+	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/types"
 )
 
 // Ant defines the fields used by a Sia Ant.
 type Ant struct {
-	apiaddr string
-	rpcaddr string
-	*exec.Cmd
+	APIAddr   string
+	RPCAddr   string
+	*exec.Cmd `json:"-"`
```

Comment:
without this, `encoding/json` encodes and attempts to decode all the types in `exec.Cmd`